bigflow/build/spec.py

```diff
@@ -84,6 +85,7 @@ def parse_project_spec(
     project_requirements_file="resources/requirements.txt",
     resources_dir="resources",
     test_framework='unittest',
+    export_image_tar=True,
```

Comment:
It wouldn't be better to use `False` as a default?

Reply:
No. Current behaviour is to export tar image, it should remain as default (at lest for the next minor version).